src/dbos-executor.ts

```diff
@@ -273,12 +273,16 @@ export class DBOSExecutor {
     }
   }
 
-  async init(...classes: object[]): Promise<void> {
+  async init(classes?: object[]): Promise<void> {
     if (this.initialized) {
       this.logger.error("Workflow executor already initialized!");
       return;
     }
 
+    if (!classes || !classes.length) {
```

Comment:
When is this called without its classes argument?

Reply:
Any time the user provides undefined (say in test or debug runtime), it gets passed into here as undefined.